i18n/en.json

```diff
@@ -1825,11 +1825,11 @@
   },
   {
     "id": "api.team.import_team.unavailable.app_error",
-    "translation": "Filesize unavilable"
+    "translation": "Filesize unavailable"
```

Comment:
"Filesize" doesn't seem very descriptive. Is there something that would be clearer here?